ODIN_II/SRC/subtractions.cpp

```diff
@@ -654,3 +668,52 @@ void clean_adders_for_sub() {
         processed_adder_list = delete_in_vptr_list(processed_adder_list);
     return;
 }
+
+/**
+ * -------------------------------------------------------------------------
+ * (function: cleanup_sub_old_node)
+ *
+ * @brief Clean up the old minus node based on the 
```

Comment:
I'd add a bit more context in the comment:  nodeo is a subtraction node that has been replaced by .... (logic in split_adder_for_sub?) ... so this function cleans up by removing nodeo and associated net etc. data from the netlist.

Reply:
For sure, thank you for such a good suggestions